cgmes/cgmes-model/src/test/java/com/powsybl/cgmes/model/CgmesOnDataSourceTest.java

```diff
@@ -37,26 +37,19 @@ class CgmesOnDataSourceTest {
 
     static Stream<Arguments> provideArguments() {
         return Stream.of(
-                Arguments.of("EQ cim14", "empty_cim14_EQ.xml", "14", true),
-                Arguments.of("EQ cim16", "empty_cim16_EQ.xml", "16", true),
-                Arguments.of("SV cim14", "empty_cim14_SV.xml", "14", false),
-                Arguments.of("cim no rdf cim16", "validCim16InvalidContent_EQ.xml", "16", false),
-                Arguments.of("cim no rdf cim14", "validCim14InvalidContent_EQ.xml", "14", false),
-                Arguments.of("rdf no cim16", "validRdfInvalidContent_EQ.xml", "16", false),
-                Arguments.of("rdf no cim14", "validRdfInvalidContent_EQ.xml", "14", false),
-                Arguments.of("rdf cim16 not cim14", "empty_cim16_EQ.xml", "14", false),
-                Arguments.of("rdf cim14 not cim16", "empty_cim14_EQ.xml", "16", false)
-        );
+                Arguments.of("EQ cim14", "empty_cim14_EQ.xml", false),
```

Comment:
Just to be clear why we have kept this test:
```suggestion
                Arguments.of("EQ cim14", "empty_cim14_EQ.xml", false), // CIM14 is not supported
```